app/locales/en/translations.js

```diff
@@ -464,6 +465,13 @@ export default {
     'username': 'Username',
     'password': 'Password',
     'myProfile': 'My Profile',
+    'manageAPITokens': 'Manage API Tokens',
+    'tokenInfo': 'API Tokens are used to programatically accesss Ilios data.  Your token is like a password - anyone who has it can make changes on your behlaf.  Read more in the documentation at <a href="{{apiDocsUrl}}">{{apiDocsUrl}}</a>',
+    'validUntil': 'When should this token stop working?',
+    'newToken': 'New Token',
+    'invalidateTokens': 'Invalidate All Tokens',
+    'invalidateTokensConfirmation': 'This will invalidate all existing API tokens and you will need to generate new ones for any applications you have.  Are you sure you want to do this?',
```

Comment:
update phrasing: invalidateTokensConfirmation': 'This will invalidate all existing API tokens and you will need to generate new ones for any applications you have which currently connect to Ilios.  Are you sure you want to do this?'